File: src/006/problem006.rs
fn square(n: i32) -> i32 {
    n * n
}

fn solve() -> i32 {
    let sum_of_the_squares: i32 = (0..101).map(square).sum();
    let square_of_the_sum = square((0..101).sum());
    square_of_the_sum - sum_of_the_squares
}

fn main() {
    let result = solve();
    println!("{}", result);
}

File: src/006/problem006.swift
extension Int {
    var squared: Int {
        return self * self
    }
}

func solve() -> Int {
    let sumOfTheSquares = (0...100).map { $0.squared }.reduce(0, +)
    let squareOfTheSum = (0...100).reduce(0, +).squared
    return squareOfTheSum - sumOfTheSquares
}

let result = solve()
print(result)
